File: SphrLibrary/Entities/OpenmHealth/PhysicalActivity.cs
using System.Runtime.Serialization;

namespace SphrLibrary.Entities.OpenmHealth
{
    /// <summary>
    /// Open mHealth 歩数データのリソースを表します。
    /// </summary>
    [DataContract()]
    [Serializable()]
    public class PhysicalActivity : OmhEntityBase<PhysicalActivityBody>
    {
        public PhysicalActivity() : base() { }

    }
}

File: References/Java/SphrLibrary-Java/lib/src/main/java/sphrlibrary/entities/openmhealth/PhysicalActivity.java
package sphrlibrary.entities.openmhealth;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.ToString;

/**
 * 歩数データ
 */
@Data
@NoArgsConstructor
@ToString(callSuper = true)
@EqualsAndHashCode(callSuper = true)
public class PhysicalActivity extends OmhEntityBase<PhysicalActivityBody> {

}
